src/main/java/use_case/signup/SignupInteractor.java

```diff
@@ -28,10 +28,10 @@ else if (!signupInputData.getPassword().equals(signupInputData.getRepeatPassword
             userPresenter.prepareFailView("Passwords don't match.");
         }
         else {
-            final User user = userFactory.create(signupInputData.getUsername(), signupInputData.getPassword());
+            final User user = userFactory.create(signupInputData.getUsername(), signupInputData.getPassword(), signupInputData.getUserID(), signupInputData.getBirthDate(), signupInputData.getFullName(), signupInputData.getEmail());
             SignupDB.save(user);
 
-            final SignupOutputData signupOutputData = new SignupOutputData(user.getName(), false);
+            final SignupOutputData signupOutputData = new SignupOutputData(user.getUsername(), false);
```

Comment:
For SignupOutputData, it should contain *all* the information signupInputData has (e.g. userID, password, email, etc)